### DISCUSSION TITLE
阈值折线图实现问题

### DISCUSSION THREAD

**Discussion by vxow** [antvis/G2#6464] [Q&A]
https://g2.antv.antgroup.com/zh/examples/general/line/#curved
以city为类型分2条线，如何实现添加一条temperature 5水位线，并且高于水位线的这线显示红色。

**Comment by zhanghengxin** [comment]
看一下    

.scale( <mark> , {
          // relations: [
          //   [(d) => d === null, '#eee'],
          //   [0, '#fff'],
          // ],
